patches/api/0475-Add-missing-fishing-event-state.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SoSeDiK <mrsosedik@gmail.com>
+Date: Wed, 1 May 2024 07:44:50 +0300
+Subject: [PATCH] Add missing fishing event state
+
+
+diff --git a/src/main/java/org/bukkit/event/player/PlayerFishEvent.java b/src/main/java/org/bukkit/event/player/PlayerFishEvent.java
+index d4001f64a7ee9d5173e9bafd9c45860cbda1fc85..03b43c88713fe5d65782cbd53dbfc86e47b281ac 100644
+--- a/src/main/java/org/bukkit/event/player/PlayerFishEvent.java
++++ b/src/main/java/org/bukkit/event/player/PlayerFishEvent.java
+@@ -165,5 +165,12 @@ public class PlayerFishEvent extends PlayerEvent implements Cancellable {
+          * in.
+          */
+         BITE
++        // Paper start - Add missing fishing event state
++        /**
++         * Called when a bobber was lured, and is now waiting to be hooked
++         * (when a "fish" starts to swim toward the bobber to bite it).
++         */
++        , LURED
++        // Paper end - Add missing fishing event state
```

Comment:
Did we check that the javadocs actually work here? The comma being in front on the same line might mess with the javadoc generation.

Reply:
Indeed, nice catch!